src/types/params.ts

```diff
@@ -24,7 +24,7 @@ export interface CreateZDAOParams {
 export interface CreateProposalParams {
   title: string;
   body: string;
-  choices: string[];
+  choices?: string[];
```

Comment:
I have a slight concern that somebody could manipulate the system by inputting the choices ["Deny", "Approve"],

Perhaps we should not allow choices to be passed for funding proposals, and default to "Approve", "Deny"

Reply:
I think so, then I will left the message in Discord and get feedback from @Eth Fish and @Brett. 
If they also agree to use default choices(Approve/Deny) for funding proposal, then I will merge into main